addons/game.controller.keyboard/addon.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="game.controller.keyboard"
+        name="IMB Model M Keyboard"
```

Comment:
Typo might want to fix

Reply:
thanks, sent upstream: https://github.com/kodi-game/controller-topology-project/pull/130